src/app/components/media/MediaFactCheck.js

```diff
@@ -37,7 +37,7 @@ const MediaFactCheck = ({ projectMedia }) => {
   const readOnly = projectMedia.is_secondary || projectMedia.suggested_main_item;
 
   const handleGoToReport = () => {
-    if (!claimDescription) {
+    if (!claimDescription || claimDescription.description.length === 0 || claimDescription.description.trim().length === 0) {
```

Comment:
No need to test for `claimDescription.description.length === 0` as `claimDescription.description.trim().length === 0` already covers that!